custom-sign-in/.eslintrc.js

```diff
@@ -1,4 +1,86 @@
 module.exports = {
-  root: true,
-  extends: '@react-native-community',
+  ignorePatterns: [
```

Comment:
Let's follow the pattern to separate eslint files into different folders, like one for root, one in src, one in test. 
Here is how we added it in [okta-react-repo](https://github.com/okta/okta-react/tree/master/test/jest).